src/components/Inputs/ColoredLabelInput.tsx

```diff
@@ -51,7 +51,7 @@ let ColoredLabelInput = ({ placeholder, onChange, value, className, id, maxLengt
   return (
     <div className={className}>
       <InputStyled
-        placeholder={placeholder}
+        label={placeholder}
```

Comment:
Why not rename the component prop to `label` as well?

Reply:
I forgot this one, thanks.